core/templates/app.constants.ts

```diff
@@ -169,6 +169,16 @@ export class AppConstants {
   };
 
   public static MISCONCEPTION_NAME_CHAR_LIMIT = 100;
+
+  // The following character limit constraints follow from
+  // android_validation_constants.py. Both have to be kept in sync.
+
+  public static MAX_CHARS_IN_TOPIC_NAME = 39;
```

Comment:
Any specific reason for this not being in constant.ts and used in both places frontend and backend?

Reply:
It had to be in a separate file so that we could make @BenHenning codeowner for it, as android dev is affected by these changes. Will move the frontend constants to constants.ts and add a test to make sure they are same according to previous comment.